lambda/models/state_machine/create_model.py

```diff
@@ -16,25 +16,62 @@
 
 from copy import deepcopy
 from typing import Any, Dict
+import json
+import os
+
+import boto3
+from botocore.exceptions import ClientError
 
 
 def handle_set_model_to_creating(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
     """Set DDB entry to CREATING status."""
     output_dict = deepcopy(event)
-    output_dict["create_infra"] = True
+    output_dict["create_infra"] = 'modelConfig' in event
```

Comment:
Ah I see what you mean now- for these, we can use the names as they're defined in the CreateModelRequest, which we can pass as direct input into the state machine. I would suggest a few things here:

1. Let's define any aws clients at the top of the file, similar to how they're done here: https://github.com/awslabs/LISA/blob/283dd8f4a2be3288e0e251a41e838e75e807fef0/lambda/models/state_machine/delete_model.py#L28-L30 (these variables are already set from the CDK definition- we can add more variables as needed)
2. For this create_infra variable, let's make it be `True` if all of AutoScalingConfig, ContainerConfig, InferenceContainer, InstanceType, and LoadBalancerConfig are not `None`.  For False, all of these need to be None, and then let's throw an exception if they're partially configured (some null, some not). Basically we have two use cases:
    1. We define the ECS infrastructure which is all of those fields, and ModelUrl must also be none
    2. We define a LiteLLM-only entry, which requires all of the others to be null, and we'd set the ModelUrl as specified in the request. For the initial revision here, let's just worry about the first case.
3.  In this step specifically, let's make sure that we set the model state to `Creating` in DDB, and we can add other details, like last modified date, the entire request blob, and anything else that seems like it'll be useful to have around. We can also add all that to the output_dict so that future states can have that info without calling out to ddb